packages/salesforcedx-sobjects-faux-generator/test/integration/sObjectDescribe.test.ts

```diff
@@ -93,6 +101,26 @@ describe('Fetch sObjects', function() {
     expect(customField.name).to.be.equal('MyCustomField__c');
   });
 
+  it('Should be able to call describeSObjectBatch on custom objects', async function() {
+    const cmdOutput = await sobjectdescribe.describeSObjectBatch(
+      process.cwd(),
+      [CUSTOM_OBJECT_NAME, CUSTOM_OBJECT2, CUSTOM_OBJECT3],
```

Comment:
I think we need better names than CustomObject1, CustomObject2,...
Anyone reading this code would not have an idea what distinguishes Object1 from Object2 and how to maintain modify in the future.

Reply:
There isn't anything that distinguishes these except their name.   I can say CustomObject1Name if you'd like.    They are simply there to manifest enough objects to show whether a batch works.